web/public/js/directives/cd-expandable-container/cd-expander/style.less

```diff
@@ -1,3 +1,6 @@
 *[cd-expander]{
   cursor: pointer;
+  .cd-expander {
+    outline: none;
```

Comment:
I know. This will get rid of the outline and make the text underline when focused via tabbing (or click).